### PR TITLE
Add muarch recipe

### PR DESCRIPTION
**Pull request by DanielBok**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/muarch/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  host:
+    - python
+    - setuptools
+    - cython >=0.29
+    - numpy >=1.14
+  run:
+    - python
+    - arch-py >=4.7
+    - copulae >=0.4
+    - numpy >=1.15
```
Comment (synapticarbors) [inline recipes/muarch/meta.yaml:29]: If your c-extensions rely on numpy, see:

https://conda-forge.org/docs/maintainer/knowledge_base.html#linking-numpy

File: recipes/muarch/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - scipy >=1.1
+    - pandas >=0.23
+
+test:
+  imports:
+    - muarch
```
Comment (synapticarbors) [inline recipes/muarch/meta.yaml:35]: add import tests for submodules:

```python
imports:
   - muarch
   - muarch.calibrate
   - muarch.datasets
   - muarch.distributions
   - muarch.funcs
   - muarch.mean
   - muarch.volatility
```

File: recipes/muarch/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e81649e7eef494d9a10e20db6f87cc4edf45b19076ec5334bfd250881cb8636f
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
```
Comment (synapticarbors) [inline recipes/muarch/meta.yaml:14]: Please use:

```
script: "{{ PYTHON }} -m pip install . -vv"
```

and add `pip` as a `host` dependency.

File: recipes/muarch/meta.yaml (hunk 1 above)
Comment (DanielBok) [reply recipes/muarch/meta.yaml]: Adopted recommendations

File: recipes/muarch/meta.yaml (hunk 2 above)
Comment (DanielBok) [reply recipes/muarch/meta.yaml:36]: Adopted recommendations

File: recipes/muarch/meta.yaml (hunk 3 above)
Comment (DanielBok) [reply recipes/muarch/meta.yaml]: Adopted recommendations

**Comment by DanielBok** [comment]
@synapticarbors , thank you for taking your time to review the pr. I've adopted all the recommendations.

**Pull request merged**